src/hooks/Storage.jsx

```diff
@@ -148,7 +145,6 @@ export const getNewPartitioning = async ({
     if (method === "AUTOMATIC") {
         await setInitializeLabelsEnabled({ enabled: true });
     }
-    await setBootloaderDrive({ drive: "" });
```

Comment:
I'll try to think a bit if not setting it here can't break creating the automatic partitioning.

Reply:
It's now done as part of the applyStorage (src/apis/storage_partitioning.js), so for automatic the behavior is the same.

Reply:
I put it wrong, I was thinking if creating the partitioning (CreatePartitioning) does not require it - vs configuring the partitioning (ConfigurePartitioning).
(Actually the concern is not regarding only automatic it was misleading in my question.) But given the value is "" I guess not.